common/commands/Cargo.toml

```diff
@@ -27,7 +27,7 @@ serde_json = { workspace = true }
 thiserror = { workspace = true }
 time = { workspace = true, features = ["parsing", "formatting"] }
 tokio = { workspace = true, features = ["sync"]}
-toml = "0.5.6"
+toml = "0.8.14"
 url = { workspace = true }
```

Comment:
this could benefit from [cargo-autoinherit](https://github.com/mainmatter/cargo-autoinherit), like [here](https://github.com/nymtech/nym/pull/4801)